README.md

```diff
@@ -5,12 +5,14 @@
 # CARMA
 ![CARMA Arch](docs/image/CARMA_web_banner_18oct18_5.png)
 
-**Cooperative Automated Research Mobility Applications (CARMA)**, CARMA is a reusable, extensible platform for controlling SAE level 1 connected, automated vehicles (CAVs). It provides a rich, generic API for third party plugins that implement vehicle guidance algorithms. It is written in Java and C++ and runs in a Robot Operating System (ROS) environment. It offers a rich application programming interface (API) for third party authors to develop guidance plugins that can plan vehicle trajectories under certain conditions. The platform can be reused on a variety of properly equipped vehicles.
+CARMA <sup>SM</sup>  advances research and development to accelerate market readiness and deployment of cooperative driving automation, while advancing automated driving technology safety, security, data, and artificial intelligence. Encourages collaboration and participation for a community of engineers and researchers to advance understanding of cooperative driving automation using open source software (OSS) and agile project management practices. 
```

Comment:
Not sure how much it matters, but I think the idea is to not have a space between the word and the "SM".  So the <sup> tag should immediately follow CARMA.  Same for the other uses below.  Probably worth changing just so there is no argument.